### DISCUSSION TITLE
lotus-miner market withdraw

### DISCUSSION THREAD

**Discussion by daveyhook** [filecoin-project/lotus#12804] [Developers/Service Providers]
Run `lotus-miner info` to view the information, "Pledge" in "Miner Balance" is 0 but "Locked" in "Market Balance" is not,how to withdraw the non-zero part in "Market Balance"?

**Comment by TippyFlitsUK** [comment]
You can't withdraw locked funds. It needs to be `Available`.

**Comment by daveyhook** [comment]
The "Locked" means collaterals for the deals to escrow, now all the sectors are expired(status:Terminated), how long will it take for these collaterals("Locked") to become "Available"?

**Reply by beck-8** [reply]
This is a status display issue. You only need to initiate a withdrawal once and all funds will be released immediately.
